website/docs/r/oracle_database_system.html.markdown

```diff
@@ -0,0 +1,174 @@
+  database_edition    = "StandardEdition"
+  database_version    = "19.27.0.0"
+  hostname            = "hostname"
+  network_anchor_id   = azurerm_oracle_network_anchor.example.id
+  resource_anchor_id  = azurerm_oracle_resource_anchor.example.id
+  shape               = "VM.Standard.x86"
+  ssh_public_keys     = ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+wWK73dCr+jgQOAxNsHAnNNNMEMWOHYEccp6wJm2gotpr9katuF/ZAdou5AaW1C61slRkHRkpRRX9FA9CYBiitZgvCCz+3nWNN7l/Up54Zps/pHWGZLHNJZRYyAB6j5yVLMVHIHriY49d/GZTZVNB8GoJv9Gakwc/fuEZYYl4YDFiGMBP///TzlI4jhiJzjKnEvqPFki5p2ZRJqcbCiF4pJrxUQR/RXqVFQdbRLZgYfJ8xGB878RENq3yQ39d8dVOkq4edbkzwcUmwwwkYVPIoDGsYLaRHnG+To7FvMeyO7xDVQkMKzopTQV8AuKpyvpqu0a9pWOMaiCyDytO7GGN you@me.com"]
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Database system. Changing this forces a new Database system to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Database system should exist. Changing this forces a new Database system to be created.
+
+* `location` - (Required) The Azure Region where the Database System should exist. Changing this forces a new Database system to be created.
+
+* `admin_password` - (Required) A strong password for SYS, SYSTEM, and PDB Admin.  Changing this forces a new Database system to be created.
+
+* `compute_count` - (Required) The number of compute servers for the Database system. Changing this forces a new Database system to be created.
+
+* `compute_model` - (Required) The compute model for Base Database Service. This is required if using the `computeCount` parameter. If using `cpuCoreCount` then it is an error to specify `computeModel` to a non-null value. The ECPU compute model is the recommended model, and the OCPU compute model is legacy. Possible values are `ECPU` , `OCPU` Changing this forces a new Database system to be created.
+
+* `database_edition` - (Required) The Oracle Database Edition that applies to all the databases on the Database system. Exadata Database systems and 2-node RAC Database systems require EnterpriseEditionExtremePerformance. Possible values are `EnterpriseEdition`, `EnterpriseEditionDeveloper`, `EnterpriseEditionExtreme`, `EnterpriseEditionHighPerformance` and `StandardEdition`. Changing this forces a new Database System to be created.
+
+* `database_version` - (Required) A valid Oracle Database version. For a list of supported versions, use the ListDbVersions operation. Changing this forces a new Database System to be created.
+
+* `hostname` - (Required) The hostname for the Database system. Changing this forces a new Database system to be created.
+
+* `license_model` - (Required) The Oracle license model that applies to all the databases on the Database system. The default is LicenseIncluded. Possible values are `BringYourOwnLicense` and `LicenseIncluded`. Changing this forces a new Database system to be created.
```

Comment:
since `license_model` is `required`, why it does not exist in `## Example Usage`?